pootle/static/js/browser/components/StatsCollapsed.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React, { PropTypes } from 'react';
+
+import Avatar from 'components/Avatar';
+import { t } from 'utils/i18n';
+
+const StatsCollapsed = React.createClass({
+    propTypes: {
+        topContributors: React.PropTypes.array.isRequired,
+    },
+
+    getScoreTextfunction: function(score) {
```

Comment:
Let's reuse the function we have in `TopContributorsTable.js` and put it in `browser/shared/utils.js`.